vignettes/articles/data_from_epireview.Rmd

```diff
@@ -0,0 +1,270 @@
+
+```{r, check-multi-row-entries}
+multi_row_entries <- duplicated(marburg_params$parameter_type) &
+  duplicated(marburg_params$id)
+multi_row_ids <- marburg_params$id[multi_row_entries]
+```
+
+```{r, subset-multi-row-entries}
+multi_row_marburg_params <-
+  marburg_params[marburg_params$id %in% multi_row_ids, ]
+```
+
+This step should be verified manually to ensure that the entries that have been selected are indeed multiple rows for the same reported epidemiological parameter. We use the first two rows of this subset table, which are the mean and standard deviation for the generation time of Marburg disease. 
+
+```{r, subset-multi-row-marburg-entry}
+marburg_gt <- multi_row_marburg_params[1:2, ]
+```
+
+We can now convert this to an `<epidist>`.
+
+```{r, convert-multi-row-entry-to-epidist}
+marburg_gt_epidist <- as_epidist(marburg_gt)
+marburg_gt_epidist
+marburg_gt_epidist$summary_stats
+```
+
+## Entries with probability distributions
+
+For this example we will load the Ebola epidemiological parameters from the {epireview} package (as there are no entries for Marburg that have parametric distributions).
+
+```{r, load-ebola}
+ebola_data <- load_epidata("ebola")
+```
+
+We will again subset the data to just use the epidemiological parameter table, and subset that table to just the delay distributions.
+
+```{r, ebola-params}
+ebola_params <- ebola_data$params
+```
+
+```{r, subset-ebola-params}
+delay_dist_rows <- grepl(
+  pattern = "Human delay",
+  x = ebola_params$parameter_type,
+  fixed = TRUE
+)
+ebola_params <- ebola_params[delay_dist_rows, ]
+ebola_params
+```
+
+We will select the 358th entry, which is a serial interval, as this entry has estimated and reported a Weibull distribution:
+
+```{r, select-ebola-entry}
+ebola_si <- ebola_params[358, ]
+ebola_si
+```
+
+We can now convert this to an `<epidist>` object.
+
+```{r, convert-to-prob-dist-epidist-single-entry}
+ebola_si_epidist <- as_epidist(ebola_si)
+ebola_si_epidist
+```
+
+With the probability distribution for the serial interval we can utilise some of the `<epidist>` methods. Here we illustrate this by checking that the `<epidist>` is parameterised, plotting the PDF and CDF, and generating 10 random numbers sampling from the distribution.
+
+```{r, is-epidist-parameterised}
+is_parameterised(ebola_si_epidist)
+```
+
+```{r, plot-epidist}
+plot(ebola_si_epidist)
+```
+
+```{r, generate-epidist}
+generate(ebola_si_epidist, times = 10)
+```
+
+## Specifying the probability distribution if unknown
+
+There may be instances where a delay distribution is reported, but either a probability distribution is not fit to the data, or it is not reported which probability distribution the parameters correspond to. In these cases, and when a parametric probability distribution is required for a particular epidemiological task then assuming a probability distribution can be useful. 
```

Comment:
Here I think it's important to clarify if you're talking about reported in epireview or in the literature because it's a bit confusing

Reply:
Resolved in e9002b980c1687e7df6040ce0f7480103683a158.